enhancements/monitoring/monitoring-stack-operator.md

```diff
@@ -0,0 +1,490 @@
+
+### Creating Dashboards
+
+The OCP Console does not (yet) support querying different Prometheus instances, making it hard to use it for creating dashboards.
+
+For this reason, the monitoring operator will optionally deploy a Grafana operator that can manage Grafana instances in the namespaces where a MonitoringStack CR is present. Managed service SREs and owners will then be able to use the CRDs provided by the Grafana operator to create their own dashboards.
+
+### Release Model
+
+We plan to follow similar “[Layered Release Cycles](https://docs.google.com/presentation/d/1b7VRJpaUcidZso6BNx4F0AmWYh5f_F1-ZmMLs4IFGLA/edit#slide=id.ge68369526c_0_0)” proposed by the Logging team.
+
+### Risks and Mitigations
+
+#### Releasing through OLM
+
+The monitoring team currently does not deploy any OpenShift component through OLM and, therefore, lacks experience in delivering software in this manner.
+
+In order to overcome this challenge, we will need to ramp up knowledge and expertise on how to test, ship and monitor software through the Operator Lifecycle Manager. Besides using the OLM documentation, we intend to leverage the fact that the OLM team is part of the OpenShift organization and we can ask for their guidance either through slack or by scheduling consulting sessions.
+
+In addition to this, several teams in the OpenShift organization already deliver their software through OLM, some of which are also in the observability realm. Specifically, the logging team has been delivering their stack in this fashion for a while, making them a good example.
+
+#### Feature duplication with CMO
+
+The Cluster Monitoring Operator has grown in features over time and the codebase contains functionality that might also be needed for the Monitoring Stack Operator. Over time, we could end up implementing the same features in two different projects.
+
+One solution which we can aim for long-term would be to deploy User Workload Monitoring as another `MonitoringStack` instance. CMO and users can still deploy UWM will be able to configure it the same way as today, only CMO would delegate the provisioning and management to MSO by creating a `MonitoringStack` resource.
+
+#### Resource usage overhead: Prometheus
+
+Deploying a single prometheus instance for each managed service does come with certain resource overhead. Even though the data will be perfectly sharded, there is still a certain fixed resource requirement for simply running Prometheus. In addition to this, sharing resources is usually more efficient because it reduces idle time.
+
+In order to evaluate the resource overhead that comes with functional sharding, we did an experiment where we deployed one Prometheus instance scraping metrics from 3 exporters (node_exporter, kube-state-metrics and kubelet) alongside 3 independent Prometheus instances, each scraping data from only one exporter.
```

Comment:
```suggestion
In order to evaluate this, we deployed one Prometheus instance scraping metrics from 3 exporters (node_exporter, kube-state-metrics and kubelet) alongside 3 independent Prometheus instances, each scraping data from only one exporter.
```

Reply:
I think it would also help to name the four prometheus instances according to the drawing below.